auth/basics/android.md

```diff
@@ -244,6 +359,14 @@ Log.i("Skygear User", "User access token: " + currentUser.getAccessToken());
 Log.i("Skygear User", "User ID: " + currentUser.getId());
 Log.i("Skygear User", "Username: " + currentUser.getUsername());
 ```
+```kotlin
+val skygear = Container.defaultContainer(this)
+val currentUser = skygear.auth.currentUser
+
+Log.i("Skygear User", "User access token: ${currentUser.getAccessToken()}")
```

Comment:
I don't think `getAccessToken` exists.

Reply:
I don't think `getUsername` exists.

Reply:
I was translating the function `getUsername` to Kotlin from the Java piece of code. I believe it's used to represent a function self written by the user, to get user (user's user) name from their side.